blog/content/post/2019-10-02-psec-2019.md

```diff
@@ -0,0 +1,125 @@
+cover: /covers/PSEC_2019.png
+author:
+  - piotr.krol
+  - lukasz.wcislo
+layout: post
+published: true
+date: 2019-10-21
+archives: "2019"
+
+tags:
+  - PSEC 2019
+  - Conference
+categories:
+  - Miscellaneous
+  - Security
+
+---
+
+PSEC 2019 enables hardware/firmware engineers, VMM/OS developers, architects,
+integrators, verifiers and senior technical staff to collaborate on
+hardware-assisted platform security and composable software supply chain
+integrity, from edge to cloud. Piotr Król, 3mdeb founder and Embedded Systems
+Consultant was one of the speakers with [the article](https://www.platformsecuritysummit.com/#krol)
+"Less-Insecure Network Edge Virtualization with Low Size, Weight and Power" and
+shared with us his thoughts and notes, which will be published here in the next  
+weeks.
+
+![Trespassers will be eaten](/img/PSEC_2019_1.jpg)
+
+The event started 1 Oct in Redmond, WA, at Microsoft Studio B with 3
+introductional videos which were meant to bring the participants into more
+exalted mood. The most interesting of the videos touched the areas of necessity
+of support for organizations like NASA that rise the level of knowledge in all
+the mankind. It was also related to the power of stories, myths, that push our
+species growth. Next all the presentations took place as planned. We'll gonna
+shorten the most interesting ones. Reader can find all the abstracts on
+the conference page.
+
+__"Guarding against physical attacks; Xbox" Tony Chen__
+
+Imagine, that your task is to create a nuclear shelter. It should be safe
+and secure but also easy accessible for anyone with proper key. And it should be
+located on the hostile territory.
+
+This was essentially the problem with one of the most attacked devices in
+history. When you are working on PC security you can assume, that your job is to
+protect the PC owner and his stuff from outer threats. You can consider him
+being a good guy. On the contrary to the Xbox the owner is a threat himself.
+It is him, who will probably try to use software and hardware tools to
+compromise it. Why should he? Because it may give him an advantage in online
+games or enable piracy.
+
+![Security architecture](/img/xbox-one.jpg)
+
+Modern gaming platforms are priced below the manufacturing cost. Companies
+want to rather earn their share with software - games designed for specific
+platform. That's why they consider eventual piracy an extremely dangerous
+threat. It may be a matter of company existence - the best example is SEGA.
+
+There are two ways to make a device recognize tampered optical disk as a genuine
+one. The first one is to make the disk absolutely identical - which can be
+obtained but is not a part of the scope of this article. The second one is to
+make a device recognize not-exactly-identical disk as the original one. This
+is the place where an physical attack can be performed.
+
+The device is controlled by the attacker. Silicon die inside the CPU has 28 nm,
```

Comment:
@lwcislo where did you get that information? https://en.wikipedia.org/wiki/Semiconductor_device_fabrication ?

Reply:
@pietrushnic https://cloud.3mdeb.com/index.php/s/dMsM4JK9Rc6BSRK